### PR TITLE
main/dconf: update to 0.32.0

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: main/dconf/APKBUILD [hunk 1]
```diff
@@ -1,32 +1,28 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dconf
-pkgver=0.26.0
+pkgver=0.32.0
 pkgrel=1
```
Comment (TBK) [inline main/dconf/APKBUILD]:
```suggestion
pkgrel=0
```

File: main/dconf/APKBUILD [hunk 2]
```diff
@@ -1,32 +1,28 @@
...
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
```
Comment (TBK) [inline main/dconf/APKBUILD]:
```suggestion
url="https://wiki.gnome.org/action/show/Projects/dconf"
```

File: main/dconf/APKBUILD [hunk 3]
```diff
@@ -1,32 +1,28 @@
...
 arch="all"
 license="LGPL-2.1"
 depends=""
```
Comment (TBK) [inline main/dconf/APKBUILD]:
```suggestion
```

File: main/dconf/APKBUILD [hunk 4]
```diff
@@ -1,32 +1,28 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dconf
-pkgver=0.26.0
+pkgver=0.32.0
 pkgrel=1
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
 arch="all"
 license="LGPL-2.1"
```
Comment (TBK) [inline main/dconf/APKBUILD]:
```suggestion
license="LGPL-2.1-or-later"
```
https://wiki.gnome.org/action/show/Projects/dconf#dconf_License

File: main/dconf/APKBUILD [hunk 5]
```diff
@@ -1,32 +1,28 @@
...
 depends=""
-makedepends="glib-dev dbus-dev docbook-xsl vala"
```
Comment (TBK) [inline main/dconf/APKBUILD]: Is bash-completion really a requirement?

File: main/dconf/APKBUILD [hunk 6]
```diff
@@ -1,32 +1,28 @@
...
-	make V=1
+		. output
+	ninja -C output
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install
+	DESTDIR="$pkgdir" ninja -C output install
 }
```
Comment (TBK) [inline main/dconf/APKBUILD]: I had a closer look why you added bash-completion . So the pkg includes a bash-completion file, you should put it in a subpkg.

```suggestion

bashcomp() {
	pkgdesc="Bash completions for $pkgname"
	install_if="$pkgname=$pkgver-r$pkgrel bash-completion"

	mkdir -p "$subpkgdir"/usr/share
	mv "$pkgdir"/usr/share/bash-completion "$subpkgdir"/usr/share
}
```

File: main/dconf/APKBUILD [hunk 7]
```diff
@@ -1,32 +1,28 @@
...
+pkgver=0.32.0
 pkgrel=1
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
 arch="all"
 license="LGPL-2.1"
 depends=""
-makedepends="glib-dev dbus-dev docbook-xsl vala"
+makedepends="glib-dev dbus-dev docbook-xsl vala gtk+3.0-dev libxml2-dev meson py3-setuptools bash-completion"
 subpackages="$pkgname-dev $pkgname-doc"
```
Comment (TBK) [inline main/dconf/APKBUILD]:
```suggestion
subpackages="$pkgname-dev $pkgname-doc $pkgname-bash-completion:bashcomp:noarch"
```

File: main/dconf/APKBUILD [hunk 8]
```diff
@@ -1,32 +1,28 @@
...
-pkgver=0.26.0
+pkgver=0.32.0
 pkgrel=1
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
 arch="all"
 license="LGPL-2.1"
 depends=""
-makedepends="glib-dev dbus-dev docbook-xsl vala"
+makedepends="glib-dev dbus-dev docbook-xsl vala gtk+3.0-dev libxml2-dev meson py3-setuptools bash-completion"
```
Comment (TBK) [inline main/dconf/APKBUILD]:
```suggestion
makedepends="glib-dev dbus-dev docbook-xsl vala gtk+3.0-dev libxml2-dev meson py3-setuptools"
```

**Comment by Cogitri** [comment]
Thank you for the review, sorry for the low quality, currently working from the TTY until GNOME works :c

**Comment by TBK** [comment]
Np :)

File: main/dconf/APKBUILD (hunk 2 above)
Comment (Cogitri) [reply main/dconf/APKBUILD]: It's https://wiki.gnome.org/Projects/dconf

File: main/dconf/APKBUILD (hunk 8 above)
Comment (Cogitri) [reply main/dconf/APKBUILD]: No, it checks for bash-completion via pkg-config, presumably to get the right dir to install it to.

**Comment by Cogitri** [comment]
Pushed an updated version.

**Pull request closed without merging**